twilight-util/src/builder/message/thumbnail.rs

```diff
@@ -31,8 +31,8 @@ impl ThumbnailBuilder {
     }
 
     /// Set the description of this thumbnail.
-    pub fn description(mut self, description: impl Into<String>) -> Self {
-        self.0.description.replace(description.into());
+    pub fn description(mut self, description: Option<String>) -> Self {
```

Comment:
I think if this function is ever called, a description for the thumbnail is to be added, so I believe passing an `Option` here is an unnecessary hassle. Perhaps we should not be changing the signature and handle the nested `Option` in the body.